swagger/api.spec.yaml

```
@@ -327,9 +327,9 @@ paths:
     get:
       tags:
         - hosts
-      summary: Get the number of tags on a host
+      summary: Get the number of tags on a host or hosts
```

Just for consistency with your other changes:
```suggestion
      summary: Get the number of tags on the hosts
```